Formula/chrome-export.rb

```diff
@@ -0,0 +1,28 @@
+class ChromeExport < Formula
+  desc "Convert Chrome’s bookmarks and history to HTML bookmarks files"
+  homepage "https://github.com/bdesham/chrome-export"
+  url "https://github.com/bdesham/chrome-export/archive/v2.0.tar.gz"
+  sha256 "ff972c3d88b030b65823388e08cd65ebd8795f0674b5581e9e8fdf3a0f4dc7ef"
+
+  depends_on "python"
```

Comment:
`depends_on "python" if MacOS.version <= :snow_leopard`
because we use the system python on recent macOS